spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/CAddTable.scala

```diff
@@ -47,12 +47,10 @@ class CAddTable[T: ClassTag, D: ClassTag](val inplace: Boolean = false)(
   private var bufferSumOutput: Tensor[D] = null
 
   private def expandWithDims(smallSize: Array[Int], otherSize: Array[Int]): Boolean = {
```

Comment:
`smallSize` and `otherSize` looks confusing.

Reply:
also the method returns boolean.  probably name as `canExpand(inputSize: Array[Int],  targetSize: Array[Int])`

Reply:
ok, changed